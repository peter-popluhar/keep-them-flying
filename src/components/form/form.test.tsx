import { render } from "@testing-library/react";
import Form from "./container";

const mockedUsedNavigate = jest.fn();

jest.mock("react-router-dom", () => ({
  ...(jest.requireActual("react-router-dom") as any),
  useNavigate: () => mockedUsedNavigate,
}));

test("renders Form", () => {
  const { container } = render(<Form />);
  expect(container).toMatchSnapshot();
});
