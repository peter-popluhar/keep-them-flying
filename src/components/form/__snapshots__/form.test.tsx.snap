// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Form 1`] = `
<div>
  <div
    class="formGrid"
  >
    <div
      class="ant-skeleton ant-skeleton-element ant-skeleton-active"
    >
      <span
        class="ant-skeleton-input"
        style="width: 100%;"
      />
    </div>
    <div
      class="ant-skeleton ant-skeleton-element ant-skeleton-active"
    >
      <span
        class="ant-skeleton-input"
        style="width: 100%;"
      />
    </div>
    <div
      class="ant-skeleton ant-skeleton-element ant-skeleton-active"
    >
      <span
        class="ant-skeleton-input"
        style="width: 100%;"
      />
    </div>
    <div
      class="ant-skeleton ant-skeleton-element ant-skeleton-active"
    >
      <span
        class="ant-skeleton-input"
        style="width: 100%;"
      />
    </div>
    <div
      class="ant-skeleton ant-skeleton-element ant-skeleton-active"
    >
      <span
        class="ant-skeleton-input"
        style="width: 100%;"
      />
    </div>
    <div
      class="ant-skeleton ant-skeleton-element ant-skeleton-active"
    >
      <span
        class="ant-skeleton-input"
        style="width: 100%;"
      />
    </div>
    <div
      class="ant-skeleton ant-skeleton-element ant-skeleton-active"
    >
      <span
        class="ant-skeleton-input"
        style="width: 100%;"
      />
    </div>
    <div
      class="ant-skeleton ant-skeleton-element ant-skeleton-active"
    >
      <span
        class="ant-skeleton-input"
        style="width: 100%;"
      />
    </div>
    <div
      class="ant-skeleton ant-skeleton-element ant-skeleton-active"
    >
      <span
        class="ant-skeleton-input"
        style="width: 100%;"
      />
    </div>
    <div
      class="ant-skeleton ant-skeleton-element ant-skeleton-active"
    >
      <span
        class="ant-skeleton-input"
        style="width: 100%;"
      />
    </div>
  </div>
</div>
`;
